package pessoal.tasks.api.infra.exceptions;

public class TarefaNaoEncontradaException extends RuntimeException{


}
